tests/core/pyspec/eth2spec/test/phase_0/sanity/test_blocks.py

```diff
@@ -363,13 +368,124 @@ def test_attester_slashing(spec, state):
     yield 'blocks', [signed_block]
     yield 'post', state
 
-    check_attester_slashing_effect(spec, pre_state, state, validator_index)
+    check_attester_slashing_effect(spec, pre_state, state, slashed_indices)
+
+
+@with_all_phases
+@spec_state_test
+def test_duplicate_attester_slashing(spec, state):
+    # Skip test if config cannot handle multiple AttesterSlashings per block
+    if spec.MAX_ATTESTER_SLASHINGS < 2:
+        return
```

Comment:
Should it be removed now? Or you want to leave it in case we change it back?

Reply:
Want to keep it in case we use other configs in the future for testing.

arguably tehre are implicit conditions on a ton of tests, but... this is fine